api/types/constants.go

```diff
@@ -688,10 +688,16 @@ const (
 	// via automatic discovery, to avoid re-running installation commands
 	// on the node.
 	VMIDLabel = TeleportInternalLabelPrefix + "vm-id"
+	// ProjectIDLabelSuffix is the identifier for adding the GCE ProjectID to an instance.
+	projectIDLabelSuffix = "project-id"
 	// ProjectIDLabel is used to identify virtual machines by GCP project
```

Comment:
This godoc wasn't changed, but the field below it is new. Please update it accordingly.